mne/transforms.py

```diff
@@ -952,7 +953,8 @@ def transform(self, source, verbose=None):
             they will be inside the convex hull formed by the original
             source points.
         verbose : bool, str, int, or None
-            If not None, override default verbose level (see mne.verbose)
+            If not None, override default verbose level (see mne.verbose and
```

Comment:
This is not addressed yet (add `verbose` to `python_reference.rst`) so none of the newly added links work